libraries/AC_PID/AC_PID.cpp

```diff
@@ -330,23 +336,17 @@ float AC_PID::update_error(float error, float dt, bool limit)
 
 // Updates the integrator based on current error and dt.
 // If `limit` is true, the integrator is only allowed to shrink to avoid wind-up.
-void AC_PID::update_i(float dt, bool limit)
+void AC_PID::update_i(float dt, bool limit, float i_scale)
```

Comment:
we should probably have a comment here explaining what i_scale is for and what callers can/cannot set it to (e.g. 0)

Reply:
Fixed